src/redux/reducers/__tests__/Connect-test.js

```diff
@@ -1034,7 +1034,10 @@ describe('Connect redux store', () => {
         },
```

Comment:
Nit: `ActionTypes.GO_BACK_OAUTH` does not expect a payload.

Reply:
Same as on the following lines:
- [ https://github.com/mxenabled/connect-widget/blob/master/src/Connect.tsx#L278]( https://github.com/mxenabled/connect-widget/blob/master/src/Connect.tsx#L278)
- [https://github.com/mxenabled/connect-widget/blob/master/src/Connect.tsx#L278](https://github.com/mxenabled/connect-widget/blob/master/src/Connect.tsx#L278)